Gemfile

```diff
@@ -21,3 +20,8 @@ group :test do
   gem 'vcr'
   gem 'webmock'
 end
+
+group :development, :test do
+  gem 'byebug'
+  gem 'pry'
+end
```

Comment:
Same here, one thing at a time. Adding gems as part of a pull request for something else is generally not a good idea.